android/app/src/main/java/com/microsoft/codepush/react/CodePush.java

```diff
@@ -379,6 +379,10 @@ public void call(DownloadProgress downloadProgress) {
                     } catch (IOException e) {
                         e.printStackTrace();
                         promise.reject(e.getMessage());
+                    } catch (CodePushInvalidUpdateException e) {
+                        e.printStackTrace();
+                        saveFailedUpdate(updatePackage);
+                        promise.reject(e.getMessage());
```

Comment:
What will this error message be? It would be great if we rejected this with a clear message ala "An update was downloaded, but ignored, because it didn't include a valid JS bundle file."